packages/cactus-plugin-ledger-connector-besu/src/main/typescript/plugin-ledger-connector-besu.ts

```diff
@@ -134,9 +139,21 @@ export class PluginLedgerConnectorBesu
       `${fnTag} options.prometheusExporter`,
     );
 
+    const web3Provider = new Web3.providers.HttpProvider(
+      this.options.rpcApiHttpHost,
+    );
+
+    this.web3 = new Web3(web3Provider);
+    this.initializeEEAClient();
```

Comment:
@travis-payne Calling this here could (as-in, it definitely will) lead to race conditions, to resolve the conundrum I opened this issue: https://github.com/hyperledger/cactus/issues/1035